tests.py

```diff
@@ -215,6 +215,31 @@ def test_lic7(self):
         points[55] = (3,1)
         self.assertFalse(c.cond_7(points, 3, 2))
 
+    #LIC 9 requires at least five points.
+    def test_lic9_less_than_five_points(self):
+        points = [(1,1),(1,1)]
+        self.assertFalse(c.cond_9(points, d.PARAMETERS))
+    
+    #LIC 9 is met if angle is larger than PI + EPSILON, and the three data points are separated by exactly C PTS and D PTS.
+    def test_lic9_angle_larger(self):
+        points = [(0,0), (1,0), (2,0), (0,5), (0,10)]
+        self.assertTrue(points, d.PARAMETERS)
```

Comment:
The angle should be between (0,0), (2,0) and (0,10), right? Because that's not the points that the condition tests.

Reply:
Yes, that was what I thought the condition was saying, or have I misunderstood?

Reply:
I think you've understood correctly, it's the condition that has the wrong indices. I've encountered similar problems in #72 and #75.

Reply:
Yes, the indexing in the condition is wrong. We need to add 1 to the indexing of the second point and 2 to the indexing of the first point. I think the same error is present in some other conditions

Reply:
Ok good, we'll have to rethink the conditions! I have created issues linked to the problems --> #83 and #84

Reply:
As William said, the fix for #84 is pretty simple.